### PR TITLE
Update redirections for Uberon

### PR DESCRIPTION
**Pull request by anitacaron**
- Issue related to the `ext.[owl,obo]` redirection https://github.com/obophenotype/uberon/issues/3031
- Issue related to the `composite-[vertebrate,metazoan].[owl,obo]` redirection https://github.com/obophenotype/uberon/issues/2800#issuecomment-1429830341

### CODE REVIEW

File: config/uberon.yml [hunk 1]
```diff
@@ -21,18 +21,24 @@ entries:
 - exact: /phenoscape-ext.owl
   replacement: http://phenoscape.svn.sourceforge.net/svnroot/phenoscape/trunk/vocab/edit/phenoscape-ext.owl
 
-- exact: /composite-vertebrate.owl
+- exact: /releases/2021-10-01/composite-vertebrate.owl
```
Comment (matentzn) [inline config/uberon.yml:24]: Why is this necessary? If you want `/composite-vertebrate.owl` to be dealt with by the fall-thru, what is the point of hard coding a standard redirect here? Why not just delete the whole goal?

Comment (anitacaron) [reply config/uberon.yml:24]: To keep this release version, no?

Comment (matentzn) [reply config/uberon.yml:24]: - exact: /releases/2021-10-01/composite-vertebrate.owl
   replacement: https://github.com/obophenotype/uberon/releases/download/v2021-10-01/composite-vertebrate.owl

Redirects the release URL ending with `/releases/2021-10-01/composite-vertebrate.owl` to the release download page of the same version.  The following piece (later on in the makefile, will do the exact same thing, unless I am completely besides me..

```
- prefix: /releases/
  replacement: https://github.com/obophenotype/uberon/releases/download/v
```

File: config/uberon.yml [hunk 2]
```diff
@@ -21,17 +21,11 @@ entries:
 - exact: /phenoscape-ext.owl
   replacement: http://phenoscape.svn.sourceforge.net/svnroot/phenoscape/trunk/vocab/edit/phenoscape-ext.owl
 
-- exact: /composite-vertebrate.owl
-  replacement: https://github.com/obophenotype/uberon/releases/download/v2021-10-01/composite-vertebrate.owl
+- exact: /ext.owl
+  replacement: https://github.com/obophenotype/uberon/releases/latest/download/uberon.owl
```
Comment (matentzn) [inline config/uberon.yml:25]: @jamesaoverton ext.owl is redirected to uberon.owl deliberately - they are now one and the same.

**Comment by jamesaoverton** [comment]
Then shouldn't `composite-[vertebrate,metazoan]` now redirect to `ext`? Or redirect them to older versions? We want to avoid dropping PURLs.

**Comment by matentzn** [comment]
That is what I tried to explain to @anitacaron [here](https://github.com/OBOFoundry/purl.obolibrary.org/pull/944#discussion_r1321504764), but maybe I am wrong here? The PURLs still work exactly the same way as they always did!

**Comment by jamesaoverton** [comment]
I'm not sure that I understand. If `exact: /composite-vertebrate.owl` is now being covered by a `prefix` entry, could `/composite-vertebrate.owl` be added to the tests for that `prefix` entry so I can see that it's not being dropped?

**Comment by matentzn** [comment]
Ok, lets do it right - I fixed it! Thanks for protecting our PURL integrity!
